pkg/datapath/config.go

```diff
@@ -58,6 +59,9 @@ type LoadTimeConfiguration interface {
 type CompileTimeConfiguration interface {
 	DeviceConfiguration
 
+	// The IPv4 gateway of the endpoint.
+	GetIPv4Gateway() net.IP
```

Comment:
Per my previous comment, this can go into `LoadTimeConfiguration` instead of `CompileTimeConfiguration` as it's just a u32 value that can be substituted at load time rather than requiring full compilation of the whole datapath to take into account.